src/modules/siptrace/siptrace.c

```diff
@@ -1332,16 +1533,226 @@ static void trace_onreply_out(struct cell *t, int type, struct tmcb_params *ps)
 	sto.stat = siptrace_rpl;
 #endif
 
-	sip_trace_store(&sto, NULL, NULL);
-	return;
+	if (info->uriState == STRACE_RAW_URI) {
+		LM_BUG("uriState must be either UNUSED or PARSED here! must be a bug! Message won't be traced!\n");
+		abort();
```

Comment:
Calling abort() will kill Kamailio, it is not possible to recover here?

Reply:
This was for debugging purposes, forgot it there. Thanks alot for this one.